src/host/VtInputThread.cpp

```diff
@@ -110,10 +110,6 @@ void VtInputThread::DoReadInput(const bool throwOnFail)
     DWORD dwRead = 0;
     auto fSuccess = !!ReadFile(_hFile.get(), buffer, ARRAYSIZE(buffer), &dwRead, nullptr);
 
-    // If we failed to read because the terminal broke our pipe (usually due
-    //      to dying itself), close gracefully with ERROR_BROKEN_PIPE.
-    // Otherwise throw an exception. ERROR_BROKEN_PIPE is the only case that
-    //       we want to gracefully close in.

```

Comment:
This comment is a copy/paste from `PtySignalInputThread`, but it doesn't actually apply to this code. Also there's only 4 uses of `_exitRequested` in total which makes this code easy to follow anyways IMO.